demos/accessible/boot.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Blockly Demos: AccessibleBlockly
+ *
+ * Copyright 2016 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Starts up Angular app.
+ * @author madeeha@google.com (Madeeha Ghori)
+ */
+
+document.addEventListener('DOMContentLoaded', function() {
+    ng.platform.browser.bootstrap(blocklyApp.AppView);
```

Comment:
One question: is this going to be different for different external devs? If not, should it be part of core AccessibleBlockly, and just end up being one more file to be included in the list of script imports?